readme.md

```diff
@@ -115,7 +242,7 @@ You can pass a query parameter `&hide=` to hide any specific stats with comma-se
 > Options: `&hide=stars,commits,prs,issues,contribs`
 
 ```md
-![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=contribs,prs)
+?username=anuraghazra&hide=contribs,prs
```

Comment:
Why did you remove the full markdown code? 🤔

Reply:
When using the GitHub action, you don't give options through the URL. Instead you give the ?param=something&... options as an input for the action. Having the markdown code will make it confusing for those using the action. Instead, I've included instructions at the top on how to customize the card for the action version and the hosted version, and I'm just mentioning the custom options below. This also makes the documentation cleaner in my opinion. No need to keep repeating the markdown code if it's already specified at the top.

Reply:
I already suspected that. I would, however, prefer having the full card generation code in here since, otherwise, I am afraid a lot of beginners will create new issues and discussions. We can document the difference in the action README. If you want, you can even add a warning/Note to make it more clear:

> **Warning**
> This is a warning.

```md
> **Warning**
> This is a warning.
```

> **Note**
> This is a warning.

```md
> **Note**
> This is a warning.
```